vowpalwabbit/search_dep_parser.cc

```diff
@@ -93,9 +93,21 @@ void initialize(Search::search &sch, size_t & /*num_actions*/, options_i &option
   all.pairs.swap(newpairs);
   all.triples.swap(newtriples);
 
+  std::vector<std::vector<namespace_index>> new_pairs(19); //19 is the length of c-style pairs array declared above
```

Comment:
Can you not just insert into all.interactions directly here as well?

Reply:
Will change this as well. Thanks for pointing out.